prediction_market_agent/tools/anvil/populate_anvil_test.py

```diff
@@ -0,0 +1,78 @@
+from eth_account import Account
+from prediction_market_agent_tooling.config import APIKeys as APIKeys_PMAT
+from prediction_market_agent_tooling.gtypes import private_key_type, xdai_type
+from prediction_market_agent_tooling.tools.contract import (
+    ContractOwnableERC721OnGnosisChain,
+)
+from prediction_market_agent_tooling.tools.hexbytes_custom import HexBytes
+from prediction_market_agent_tooling.tools.web3_utils import send_xdai_to, xdai_to_wei
+from web3 import Web3
+
+from prediction_market_agent.agents.microchain_agent.nft_treasury_game.constants_nft_treasury_game import (
+    NFT_TOKEN_FACTORY,
+)
+from prediction_market_agent.db.agent_communication import send_message
+from prediction_market_agent.tools.message_utils import compress_message
+
+if __name__ == "__main__":
+    agent1 = Account.from_key(
+        "0x57d188552ae3933dfb4bbbff7f9e6a31c396a2b0d74a6c464e6035396bee4626"
```

Comment:
@gabrielfior All these private keys must be ditched and new ones created for our agents. 

Afterward, please update `wallet_address` [properties in our classes](https://github.com/gnosis/prediction-market-agent/blob/849a345357da367203eab23505c0ca84314a5524/prediction_market_agent/agents/microchain_agent/nft_treasury_game/deploy_nft_treasury_game.py#L127) and private keys in GCP secrets.

If I remember correctly, we don't hard-code their public keys anywhere else except for the `wallet_address` property, so all should be working automatically after this.

There are 4 real private keys out of 7, were others pushed as well at some point in time? If so, they must be updated as well.

(seems like so far no one abused these keys, so you can also send any funds left to these new agents)

Reply:
I'll take a look how to prevent this into the future.

Reply:
Just remembered that Hugo has public keys hardcoded in Dune as well. Ping him about the change as well please 🙏

Reply:
My bad - already did the emergency actions and NFT keys + small funds are safe.